### PR TITLE
add group attendee

### PR DESCRIPTION
**Pull request by phannh58**

### CODE REVIEW

File: app/services/events/create_service.rb [hunk 1]
```diff
@@ -31,6 +39,21 @@ def modify_repeat_params
       @event.repeat_ons_attributes = []
     end
 
+    def build_attendee_group_details
+      @event.attendees.each do |attendee|
```
Comment (dieunb) [inline app/services/events/create_service.rb]:
```
group_attendee = GroupAttendee.new name: @params[:group_attendee_name], user_id: @user.id
group_attendee.attendees = @event.attendees
group_attendee.save!
```

File: app/services/events/create_service.rb [hunk 2]
```diff
@@ -31,6 +39,21 @@ def modify_repeat_params
       @event.repeat_ons_attributes = []
     end
 
+    def build_attendee_group_details
```
Comment (dieunb) [inline app/services/events/create_service.rb]: `def make_group_attendee`

File: app/services/events/create_service.rb [hunk 3]
```diff
@@ -31,6 +39,21 @@ def modify_repeat_params
...
+      @event.attendees.each do |attendee|
+        AttendeeGroupDetail.create! attendee_id: attendee.id, group_attendee_id: @group_attendee.id
+      end
+    end
+
+    def build_attendee_from_params_group_attendee
+      @params[:group_attendee][:names].each do |group_attendee|
```
Comment (dieunb) [inline app/services/events/create_service.rb]:
```
attendees = Attendee.joins("group_attendee_details").where group_attendee_details: {group_attendee_id: @params[:group_attendee][:ids]
@event.attendees = attendees 
```

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -67,4 +67,15 @@ def go_back_link
       cookie.split("\;").last
     end
   end
+
+  def load_attendee group
+  (attendees = group.attendees
+    content_tag(:ol) do
+      attendees.each do |attendee|
+        concat(content_tag(:li) do
+           attendee.email
```
Comment (TaoLuyen) [inline app/helpers/application_helper.rb]: ident

File: app/assets/javascripts/attendee_action.js [hunk 5]
```diff
@@ -44,15 +45,44 @@ $(document).on('ready', function() {
     return $('<li data-attendee-email="' + item.email + '">').append('<div class="ui-item"><i class="fa fa-user" aria-hidden="true"></i> <a data-user-id=' + item.user_id + '>' + item.email + '</a></div>').appendTo(ul);
   };
 
+  $('input[type="checkbox"]#group_attendee_ids').change(function(){
```
Comment (TaoLuyen) [inline app/assets/javascripts/attendee_action.js]: $('input[type="checkbox"]#group_attendee_ids').change(function() {

File: app/assets/javascripts/attendee_action.js [hunk 6]
```diff
@@ -44,15 +45,44 @@ $(document).on('ready', function() {
...
+    var list_email = $(this).siblings().last().attr("data-content");
+    var array_email = $(list_email).text().slice(0,-1).split(',');
+    if (this.checked){
```
Comment (TaoLuyen) [inline app/assets/javascripts/attendee_action.js]: if (this.checked) {

File: app/assets/javascripts/attendee_action.js [hunk 7]
```diff
@@ -44,15 +45,44 @@ $(document).on('ready', function() {
...
+      for(i = 0; i < array_email.length; i ++){
+        var attendee = attendeeTemplate(array_email[i]);
+        $('#list-attendee').append(attendee);
+        $('#group_attendee_name').removeAttr('disabled');
+      }
+    } else {
+      for(i = 0; i < array_email.length; i ++){
```
Comment (TaoLuyen) [inline app/assets/javascripts/attendee_action.js]: nt

File: app/assets/javascripts/attendee_action.js [hunk 8]
```diff
@@ -44,15 +45,44 @@ $(document).on('ready', function() {
...
     $('#list-attendee').append(attendee);
+    $('#group_attendee_name').removeAttr('disabled');
   });
 
   $('#list-attendee').on('click', '.remove-attendee', function(){
     $(this).parents('.attendee').remove();
+    removeGroupAttendeeName();
   });
 
+  function removeGroupAttendeeName(){
```
Comment (TaoLuyen) [inline app/assets/javascripts/attendee_action.js]: nt

File: app/assets/javascripts/event.js [hunk 9]
```diff
@@ -252,6 +252,17 @@ $(document).on('ready', function() {
     $('.dialog-repeat-event').hide();
   }
 
+  $('[data-toggle="popover"]').popover();
+
+  $('input[type="checkbox"]#group_attendee_name').change(function() {
+    debugger
```
Comment (TaoLuyen) [inline app/assets/javascripts/event.js]: ???

File: app/assets/stylesheets/custom.scss [hunk 10]
```diff
@@ -570,3 +570,15 @@ select {
   background-position: 100% 50% !important;
   background-repeat: no-repeat !important;
 }
+
+.checkbox{
```
Comment (TaoLuyen) [inline app/assets/stylesheets/custom.scss]: .checkbox {

**Pull request merged**